src/assets/locales/ua.json

```diff
@@ -0,0 +1,446 @@
+        "app-info.privacy-and-security-advice": "Щоб отримати поради щодо захисту вашої інформаційної панелі, ви можете звернутися до розділу",
+        "app-info.privacy-and-security-advice-link": "Документи керування",
+        "privacy-and-security-security-issue": "Якщо ви виявили потенційну проблему з безпекою, повідомте про це до розділу",
+        "privacy-and-security-security-policy": "Політика безпеки",
+        "license": "Ліцензія",
+        "license-under": "Ліцензія згідно",
+        "licence-third-party": "Ліцензії на сторонні модулі див.",
+        "licence-third-party-link": "Юридична інформація",
+        "list-contributors": "Повний список співавторів та подяки див.",
+        "list-contributors-link": "Кредити",
+        "version": "Версія"
+    },
+    "config": {
+        "main-tab": "Головне меню",
+        "view-config-tab": "Переглянути конфігурацію",
+        "edit-config-tab": "Редагувати конфігурацію",
+        "custom-css-tab": "Власні стилі",
+        "heading": "Параметри конфігурації",
+        "download-config-button": "Переглянути / Експортувати конфігурацію",
+        "edit-config-button": "Редагувати конфігурацію",
+        "edit-css-button": "Редагувати власний CSS",
+        "cloud-sync-button": "Увімкнути хмарну синхронізацію",
+        "edit-cloud-sync-button": "Редагувати хмарну синхронізацію",
+        "rebuild-app-button": "Перебудувати програму",
+        "change-language-button": "Змінити мову програми",
+        "reset-settings-button": "Скинути локальні налаштування",
+        "disabled-note": "Деякі функції конфігурації були відключені вашим адміністратором",
+        "small-screen-note": "Ви використовуєте дуже маленький екран, і деякі екрани в цьому меню можуть бути не найкращими",
+        "app-info-button": "Інформація про програму",
+        "backup-note": "Рекомендовано зробити резервну копію конфігурації перед внесенням змін.",
+        "reset-config-msg-l1": "Це видалить усі налаштування користувача з локального сховища, але не вплине на ваш файл 'conf.yml'.",
+        "reset-config-msg-l2": "Ви повинні спочатку створити резервну копію будь-яких змін, які ви внесли локально, якщо ви хочете використовувати їх у майбутньому.",
+        "reset-config-msg-l3": "Ви впевнені, що бажаєте продовжити?",
+        "data-cleared-msg": "Дані успішно видалено",
+        "actions-label": "Дії",
+        "copy-config-label": "Копіювати конфігурацію",
+        "data-copied-msg": "Конфігурацію скопійовано до буфера обміну",
+        "reset-config-label": "Скинути конфігурацію",
+        "css-save-btn": "Зберегти зміни",
+        "css-note-label": "Примітка",
+        "css-note-l1": "Вам потрібно буде оновити сторінку, щоб ваші зміни набули чинності.",
+        "css-note-l2": "Заміни стилів зберігаються лише локально, тому рекомендується зробити копію вашого CSS.",
+        "css-note-l3": "Щоб видалити всі спеціальні стилі, видаліть вміст та натисніть "Зберегти зміни"",
```

Comment:
Looks like you've got an extra `"` here, which is making it invalid JSON

Reply:
Replace with this, and we should be good to go :)

```diff
- "css-note-l3": "Щоб видалити всі спеціальні стилі, видаліть вміст та натисніть "
-		Зберегти зміни "",
+ "css-note-l3": "Щоб видалити всі спеціальні стилі, видаліть вміст та натисніть Зберегти зміни",
```

Reply:
Actually was only a quick change, so I've updated it in 344a10916637373fd8010d06ecb9fd4e5222ad2a :)

Reply:
Thank you @Lissy93 and sorry for my mistake. Have a great day 🥰